lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -605,14 +605,26 @@ _.assign(Builders.prototype, {
      * Constructs an array of "sample" responses (compatible with a V2 collection)
      * from a Postman Collection V1 Request.
      *
+     * If response ordering via `responses_order` field is present,
+     * ensure the ordering is respected while constructing responses array.
+     *
      * @param {Object} requestV1 - The v1 request object to extract response information from.
      * @returns {Object[]} - The list of v2 response definitions.
      */
     response: function (requestV1) {
-        var self = this;
+        var self = this,
+            responses = _.get(requestV1, 'responses', []),
+            responsesCache = _.keyBy(responses, 'id'),
+            responses_order = _.get(requestV1, 'responses_order', []);
+
+        // If ordering of responses is not available
+        // Create a default ordering using the `responses` field
+        if (!responses_order || (responses_order && responses_order.length === 0)) {
```

Comment:
Same as:
```suggestion
        if (!(responses_order && responses_order.length)) {
```

But it's fine.